cmd/test/debugArwen.go

```diff
@@ -9,6 +9,20 @@ import (
 )
 
 func main() {
+	if len(os.Args) != 2 {
+		panic("One argument expected - the path to the json test.")
+	}
+	jsonTestPath := os.Args[1]
+
+	err := controller.RunSingleIeleTest(
```

Comment:
Maybe rename `RunSingleIeleTest()` to the generic `RunSingleVMTest()`?

Reply:
good point, I changed the one with many tests but forgot about this one